src/components/PatientProfile/PatientProfilePage.tsx

```diff
@@ -194,11 +194,17 @@ const PatientProfilePage = ({
   };
 
   const action = () => (
-    <>
+    <Link
+      to={{
+        pathname: `/events/${eventId}/ccps/${ccpId}/patientDetails/${patientId}`,
+        state: { modal: true },
+      }}
+      className="link"
+    >
       <Button style={{ color: Colours.SnackbarButtonBlue }}>
         View Patient Details
       </Button>
```

Comment:
is there a way to remove this `Button` tag entirely and replace it with the `Link` tag (and style it accordingly)

Reply:
opted for cleaner onClick event